package.json

```diff
@@ -64,13 +58,19 @@
     "expect.js": "^0.3.1",
     "fs-extra": "3.0.1",
     "glob": "^7.1.1",
+    "husky": "^0.14.3",
     "istanbul": "^0.4.5",
     "knex": "0.x",
+    "lint-staged": "^4.2.3",
```

Comment:
No thanks to lint-staged and husky, especially given @koskimas git-hunk commit flow.